Casks/e/egovframedev.rb

```diff
@@ -12,7 +12,7 @@
 
   livecheck do
     url "https://www.egovframe.go.kr/home/sub.do?menuNo=39"
-    regex(/개발자용\s개발환경\s(\d+(?:\.\d+)+).*/i)
+    regex(/개발자용\s개발환경\s(\d+(?:\.\d+)+)\s+(?!Beta)/i)
```

Comment:
```suggestion
    regex(/개발자용\s+개발환경\s+v?(\d+(?:\.\d+)+)\s+(?!Beta)/i)
```

Unrelated, but might as well make some more adjustments.